tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -155,6 +155,8 @@ Container on Open Network And Port Forwarding Persist After Reboot
 
     Log To Console  Create Port Groups For Container network
     ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vswitch vSwitchLAN open-net
+    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Add VC Distributed Portgroup  test-ds  open-net
```

Comment:
Looks like this will fail if test-ds and open-net already exist, which would be the case if the test fails with FAST_FAIL set because we do not have the removal as part of the test cleanup.

Reply:
We aren't actually parsing the response here, so it can and will fail if it is already created, but doesn't affect anything.

Reply:
@andrewtchin ?